ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,6 +1202,12 @@ literals:
 - dimmer_percentage_command: # Should be adjusted to brightness_percentage_command
     fixed_min: 0.0
     fixed_max: 100.0
+- aisle_brightness_percentage_command:
```

Comment:
What does `aisle` represent in your use-case? It's typically used in server room applications, but I'm assuming it means something different here

Reply:
These open offices usually have two rows of workplaces: One closer to the windows / facade and one row closer to the aisle or the door where one enters the office. So in this context aisle represents the hallway or corridor (thinking of it, "hallway* would maybe be the better subfield for this...)

Reply:
Ok thank you for clarifying - it's ok to leave this as `aisle`